src/components/tab-bar/tab-bar.tsx

```diff
@@ -26,7 +26,7 @@ export class TabBar {
      * List of tabs to display
      */
     @Prop()
-    public tabs: Tab[];
+    public tabs: Tab[] = [];
```

Comment:
I tested the PR by updating one of the examples to initially have supply `undefined` and then change to an array of tabs when I clicked a button. The component still crashed, before it got to `tabsChanged`. So I searched the file for `.map`, and there's another map function in the render function (line 78). By adding a default value here too, the crashing is resolved.

I tried it with only this default value, and no default values for the `tabsChanged` function, and it still didn't crash, even when the tabs where updated. But I think it's more prudent to have default values in both places, just in case.